apps/backend/db_patches/0158_UpdateOldInstrumentPickerAnswers.sql

```diff
@@ -0,0 +1,34 @@
+DO
+$$
+BEGIN
```

Comment:
I will try to run this against our develop database and get back with approval if everything works.

Reply:
Hey @Bhaswati1148 after running this migration locally I noticed that it works well when we had value just as number for example:
this answer:
```typescript
{
  "value": 4
}
``` 
was converted to this: 
```typescript
{
  "value": {
    "instrumentId": "4",
    "timeRequested": null
  }
}
``` 

but it seems to have a problem when converting some other cases like for example:
this one:
```typescript
{
  "value": [4]
}
``` 
was converted to this:
```typescript
{
  "value": {
    "instrumentId": "[4]",
    "timeRequested": null
  }
}
``` 
which I think it is wrong. It should be something like:
```typescript
{
  "value": [{
    "instrumentId": "4",
    "timeRequested": null
  }]
}
``` 
And then another example is where we have:
```typescript
{
  "value": [19, 20, 21]
}
``` 
converted to:
```typescript
{
  "value": {
    "instrumentId": "[19, 20, 21]",
    "timeRequested": null
  }
}
``` 
which I also think it is wrong and it should be:
```typescript
{
  "value": [{
    "instrumentId": "19",
    "timeRequested": null
  },
  {
      "instrumentId": "20",
      "timeRequested": null
   },
  {
      "instrumentId": "21",
      "timeRequested": null
  }]
}
```